examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,450 @@
+   For demonstration let's create 3 accounts: `<CONTRACT_ACCOUNT>`, `<ALICE_ACCOUNT>`, `<BOB_ACCOUNT>`
+   ```shell
+   $ near create-account <CONTRACT_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet --masterAccount <MASTER_ACCOUNT_NAME>.testnet --initialBalance 10
+   $ near create-account <ALICE_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet --masterAccount <MASTER_ACCOUNT_NAME>.testnet --initialBalance 10
+   $ near create-account <BOB_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet --masterAccount <MASTER_ACCOUNT_NAME>.testnet --initialBalance 10
+   ```
+
+   In the next sections, we will refer to the `<CONTRACT_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet` as `<CONTRACT_ACCOUNT>`, 
+   to the `<ALICE_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet` as `<ALICE_ACCOUNT>`, and to the `<BOB_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet` as `<BOB_ACCOUNT>` for simplicity.
+
+2. **Compile Contract to wasm file**
+   For compiling the contract, just run the `access_controllable_base/build.sh` script. The target file with compiled contract will be `../target/wasm32-unknown-unknown/release/access_controllable_base.wasm`
+
+   ```shell
+   $ cd access_controllable_base
+   $ ./build.sh
+   $ cd ..
+   ```
+
+3. **Deploy and init a contract**
+   ```shell
+   $ near deploy --accountId <CONTRACT_ACCOUNT> --wasmFile ../target/wasm32-unknown-unknown/release/access_controllable_base.wasm --initFunction new --initArgs '{}'
+   ```
+
+## Example of using the contract with access control plugin
+### Calling the methods with access control
+For using the method, `level_a_incr` you should be a member of GroupA.  Alice is not a member of any group, so she can't use this method.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <ALICE_ACCOUNT>
+ERROR
+$ near view get_counter
+0
+```
+
+Let's make Alice a member of GroupA.
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_grant_role '{"role": "GroupA", "account_id": "<ALICE_ACCOUNT>"}' --accountId <CONTRACT_ACCOUNT>
+```
+
+Now Alice is a member of GroupA and can call the `level_a_incr` method
+```shell
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <ALICE_ACCOUNT>
+$ near view get_counter
+1
+```
+
+As well as calls the `level_ab_incr` method, which allowed for both GroupA and GroupB members.
+```shell
+$ near call <CONTRACT_ACCOUNT> level_ab_incr --accountId <ALICE_ACCOUNT>
+$ near view get_counter
+2
+```
+
+### Admin of the group not a member of the group
+Note, the admin of the group may not be a member of this group. For example, the `<CONTRACT_ACCOUNT>` is a super admin, but he
+can't execute the `level_a_incr` method. 
+
+```shell
+$ near view get_counter
+2
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <CONTRACT_ACCOUNT>
+ERROR
+$ near view get_counter
+2
+```
+
+### A few super admins
+There may be two or more super admins. 
+By default, there are no interfaces that allow you to add or remove super admins. 
+You can add the corresponding functions to the contract as follows:
```

Comment:
I'd rephrase it a bit, something along those lines:

```suggestion
### Multiple super admins
There may be multiple super admins. 
By default, only `acl_init_super_admin` is exposed on the contract. To add more super admins or to remove super admins, internal functions can be used. 
To make this functionality publicly available, you could add functions like the following to your contract:
```

Reply:
fix https://github.com/aurora-is-near/near-plugins/pull/34/commits/1a7a34b88d2b4513d1de096b408ef5fc17e46209